wp_api/src/api_client.rs

```diff
@@ -1,66 +1,21 @@
-use crate::request::{
-    endpoint::{
-        application_passwords_endpoint::{
-            ApplicationPasswordsRequestBuilder, ApplicationPasswordsRequestExecutor,
+use crate::{
+    request::{
+        endpoint::{
+            application_passwords_endpoint::ApplicationPasswordsRequestExecutor,
+            plugins_endpoint::PluginsRequestExecutor,
+            post_types_endpoint::PostTypesRequestExecutor,
+            posts_endpoint::PostsRequestExecutor,
+            site_settings_endpoint::SiteSettingsRequestExecutor,
+            users_endpoint::UsersRequestExecutor,
+            wp_site_health_tests_endpoint::WpSiteHealthTestsRequestExecutor,
+            ApiBaseUrl,
         },
-        plugins_endpoint::{PluginsRequestBuilder, PluginsRequestExecutor},
-        post_types_endpoint::{PostTypesRequestBuilder, PostTypesRequestExecutor},
-        posts_endpoint::{PostsRequestBuilder, PostsRequestExecutor},
-        site_settings_endpoint::{SiteSettingsRequestBuilder, SiteSettingsRequestExecutor},
-        users_endpoint::{UsersRequestBuilder, UsersRequestExecutor},
-        wp_site_health_tests_endpoint::{
-            WpSiteHealthTestsRequestBuilder, WpSiteHealthTestsRequestExecutor,
-        },
-        ApiBaseUrl,
+        RequestExecutor,
     },
-    RequestExecutor,
 };
 use crate::{ParsedUrl, WpAuthentication};
 use std::sync::Arc;
 
-#[derive(Debug, uniffi::Object)]
-struct UniffiWpApiRequestBuilder {
-    inner: WpApiRequestBuilder,
-}
-
-#[uniffi::export]
-impl UniffiWpApiRequestBuilder {
-    #[uniffi::constructor]
-    pub fn new(site_url: Arc<ParsedUrl>, authentication: WpAuthentication) -> Self {
-        Self {
-            inner: WpApiRequestBuilder::new(site_url, authentication),
-        }
-    }
-}
-
-#[derive(Debug)]
-pub struct WpApiRequestBuilder {
-    application_passwords: Arc<ApplicationPasswordsRequestBuilder>,
-    plugins: Arc<PluginsRequestBuilder>,
-    post_types: Arc<PostTypesRequestBuilder>,
-    posts: Arc<PostsRequestBuilder>,
-    site_settings: Arc<SiteSettingsRequestBuilder>,
-    users: Arc<UsersRequestBuilder>,
-    wp_site_health_tests: Arc<WpSiteHealthTestsRequestBuilder>,
-}
-
-impl WpApiRequestBuilder {
-    pub fn new(site_url: Arc<ParsedUrl>, authentication: WpAuthentication) -> Self {
-        let api_base_url: Arc<ApiBaseUrl> = Arc::new(site_url.inner.clone().into());
-        macro_helper::wp_api_request_builder!(
-            api_base_url,
-            authentication;
-            application_passwords,
-            plugins,
-            post_types,
-            posts,
-            users,
-            site_settings,
-            wp_site_health_tests
-        )
-    }
-}
```

Comment:
I removed these codes because they are not used anywhere. If you think they are important, I can add them back.

Reply:
It's not unused code. It's part of the public API that supports manual request building and execution as showcased by the now removed integration test. Please keep in mind that this library is supposed to be used by clients other than our own.

Also, even if we were going to remove something like this, it shouldn't be part of a completely unrelated PR as that buries the information and makes it harder to bring back.

Reply:
I have reverted this change.